deltachat-ios/Controller/ChatListController.swift

```diff
@@ -89,8 +89,7 @@ class ChatListController: UITableViewController {
         let msg = dcContext.newMessage(viewType: DC_MSG_TEXT)
         msg.text = String.localized("update_1_28_android") + "\n\n" + String.localized("update_1_28_ios_extra_line")
         dcContext.addDeviceMessage(label: "update_1_28a_ios", msg: msg)
-
-        viewModel.refreshData()
```

Comment:
this call has been removed. It calls dcContext.getChatlist() internally and is executed on the main thread.